### ISSUE TITLE
Missing CSS code hints for logical properties & values

### ISSUE THREAD

**Issue by aderpio** [adobe/brackets#14788]
Working on a RLT web app i noticed that Brackets CSS code hints seem to be missing almost all the CSS logical properties used in rtl development. For example CSSProperties.json contains the `margin top/right/left/bottom` but not the:
```
margin-block
margin-block-end
margin-block-start
margin-inline
margin-inline-end
margin-inline-start
```
All CSS logical properties are to be found at:
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Logical_Properties

Should I attempt a pull request updating with all those properties/values?

Using Windows 7 x64, Brackets v1.13

**Comment by shubhsnov** [comment]
I do see that these values are experimental as of yet and might not even be supported on all browsers (hover over the icon next to the property). The page itself discourages the use in production code. 

By giving them as hints, we in spirit encourage them to use these non-standard properties.
@swmitra what do you say?

As for pull request, since this is a very targetted ask and might be extremely relevant to your use case,  I think it better to have this as a separate extension which can have these extra hints. You can upload it to the extension registry for others to use as well. This will also allow you to disable the feature and revert back to using the standard hints as and when you please.

**Comment by aderpio** [comment]
You are right, these properties & values are w3c editor's draft.  Perhaps an extension would be the best alternative at this point in time. However, given that: 
- All logical box model and the new width/height properties work in all browsers (save Edge),
- LTR/RTL UI developers would probably use these properties already,
- Policing code hints based only on what's "official" may not be a code editor's main job,

we could make the case for giving this another thought or at least keeping an eye on it :) There are many CSS properties/values that are not fully supported but used plenty in production in various cases. For example content for Android WebView.

**Comment by shubhsnov** [comment]
@narayani28 @swmitra Any thoughts on this?

**Comment by sprintr** [comment]
Not in favor of adding experimental properties to the hints. I guess making an extension would be a better approach.
